src/frontend/templates/run_stats_head.html

```diff
@@ -0,0 +1,440 @@
+{% load i18n %}
+{% load custom_translation %}
+
+<link type="text/css" href="/static/css/tooltips.css" rel="stylesheet" />
+<link type="text/css" href="/static/css/elo-graph.css" rel="stylesheet" />
+<script type="text/javascript" src="/static/js/d3.v6.min.js"></script>
+
+{% url 'networks-for-elo-list' as networks_for_elo_list_url %}
+<script>
+var api_url = "{{networks_for_elo_list_url|escapejs}}";
+var run_name = "{{run.name|escapejs}}";
+
+function makeGraph() {
+  d3.json(api_url + "?run__name=" + run_name).then(function(networks) {
+    // Extract unique network sizes
+    var networkSizes = new Set(networks.map(network => network["network_size"]));
+    // Natural numeric sort
+    var collator = new Intl.Collator(undefined, {numeric: true, sensitivity: 'base'});
+    var compare = function(a,b) {
+      aIsRandom = a.toLowerCase() == "random";
+      bIsRandom = b.toLowerCase() == "random";
+      if(aIsRandom && bIsRandom)
+        return 0;
+      if(aIsRandom)
+        return -1;
+      if(bIsRandom)
+        return 1;
+      return collator.compare(a,b);
+    };
```

Comment:
Another solution is having django outputing directly data. 
I think you solution is cleaner.

Reply:
Thanks.